rpm/qm.spec

```diff
@@ -21,6 +29,10 @@
 %endif
 
 Name: qm
+# Set different Epochs for copr and koji
+%if %{with copr}
+Epoch: 101
+%endif
```

Comment:
Hm, what's the goal of playing with the epoch here? there are fairly high costs associated with epochs :/